### ISSUE TITLE
Allow to define custom icon uri

### ISSUE THREAD

**Issue by iLLiCiTiT** [ynput/ayon-applications#21]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Please describe the feature you have in mind and explain what the current shortcomings are?

Right now we do allow only icons that are available inside applications addon, so custom applications can set custom icon.

That is due to recent changes in applications addon and movement out of ayon-core.

### How would you imagine the implementation of the feature?

Because webactions are comming, we should support custom uri for icon in additional applications. I guess the icons could be supplied by an addon,

Question that might rise is how to make it dynamic enough, the final uri could be like `https://my.ayon.server/addons/myAddon/1.1.1/private/icons/myApp.png` but the template probably should be `{ayon_url}/addons/myAddon/{addon_versions[myAddon]}/private/icons/myApp.png` so it can be used dynamically, ayon_url would be filled with whatever client needs and `addon_versions` would be filled with addon versions based on addons available in bundle that is used.

### Are there any labels you wish to add?

- [X] I have added the relevant labels to the enhancement request.

**Comment by BigRoy** [comment]
@iLLiCiTiT is this now solved?

**Comment by iLLiCiTiT** [comment]
No. This one would resolve it https://github.com/ynput/ayon-applications/pull/136
